Doesn't build with USE_SYSTEM_VORBIS=1

Build fails with:
```
LD build/release-linux-x86_64/quake3e.x64
ld: build/release-linux-x86_64/client/snd_codec_ogg.o: in function `S_OGG_CodecCloseStream':
snd_codec_ogg.c:(.text+0xe3): undefined reference to `ov_clear'
ld: build/release-linux-x86_64/client/snd_codec_ogg.o: in function `S_OGG_CodecOpenStream':
snd_codec_ogg.c:(.text+0x172): undefined reference to `ov_open_callbacks'
ld: snd_codec_ogg.c:(.text+0x186): undefined reference to `ov_seekable'
ld: snd_codec_ogg.c:(.text+0x193): undefined reference to `ov_streams'
ld: snd_codec_ogg.c:(.text+0x1a3): undefined reference to `ov_info'
ld: snd_codec_ogg.c:(.text+0x1b5): undefined reference to `ov_pcm_total'
ld: snd_codec_ogg.c:(.text+0x204): undefined reference to `ov_clear'
```

--- a/Makefile
+++ b/Makefile
@@ -237,8 +237,8 @@ ifneq ($(call bin_path, $(PKG_CONFIG)),)
     OGG_LIBS ?= $(shell $(PKG_CONFIG) --silence-errors --libs ogg || echo -logg)
   endif
   ifeq ($(USE_SYSTEM_VORBIS),1)
-    VORBIS_CFLAGS ?= $(shell $(PKG_CONFIG) --silence-errors --cflags vorbis || true)
-    VORBIS_LIBS ?= $(shell $(PKG_CONFIG) --silence-errors --libs vorbis || echo -lvorbis)
+    VORBIS_CFLAGS ?= $(shell $(PKG_CONFIG) --silence-errors --cflags vorbisfile || true)
+    VORBIS_LIBS ?= $(shell $(PKG_CONFIG) --silence-errors --libs vorbisfile || echo -lvorbisfile)
   endif
 endif
 
@@ -267,7 +267,7 @@ ifeq ($(USE_SYSTEM_OGG),1)
 endif
 ifeq ($(USE_SYSTEM_VORBIS),1)
   ifeq ($(VORBIS_LIBS),)
-    VORBIS_LIBS = -lvorbis
+    VORBIS_LIBS = -lvorbisfile
   endif
 endif
 

